js/apps/admin-ui/src/authentication/components/FlowRow.tsx

```diff
@@ -102,9 +102,7 @@ export const FlowRow = ({
                 />
               </DataListCell>,
               <DataListCell key={`${execution.id}-config`}>
-                {execution.configurable && (
```

Comment:
Are you sure this condition can be removed? Or is this some debugging code?

Reply:
Hi @jonkoops, this was originally used to hide the execution configuration if the authenticator is not marked as configurable. however, this implementation sets a default configuration on all authenticators for the authenticator reference, so this check was moved to inside the ExecutionConfigModal component to line 77, and will fetch the config description from keycloak if the execution is set as configurable or default to an empty list, then add the default authenticator reference config to that list of configuration items.